src/AutoRest.CSharp/Common/Output/Builders/SerializationBuilder.cs

```diff
@@ -296,7 +299,7 @@ public JsonObjectSerialization BuildJsonObjectSerialization(ObjectSchema objectS
                     var schemaProperty = objectTypeProperty.SchemaProperty;
```

Comment:
This line can be removed, `schemaProperty` is not in use anymore.

Reply:
removed